datumaro/cli/contexts/project/__init__.py

```diff
@@ -677,7 +677,7 @@ def transform_command(args):
     extra_args = {}
     if hasattr(transform, 'parse_cmdline'):
         extra_args = transform.parse_cmdline(args.extra_args)
-
+    
```

Comment:
```suggestion

```

Reply:
OK, I will consider making a new plugin type.

Reply:
Can you tell me detailed guideline how to make new plugin type(DatasetValidator)

Reply:
Define and add it to the `Environment` and `CliPlugin` classes. Maybe, you'd better do this in another PR, to avoid blocking this one.

Reply:
Do you mean handling three requested feedback above and handling new plugin type in additional PR?

Reply:
I will handle new plugin type in another PR!